pkg/aws/eni/routing/routing_test.go

```diff
@@ -51,6 +51,19 @@ func (e *ENIRoutingSuite) TestConfigure(c *C) {
 	runFuncInNetNS(c, func() { runConfigureThenDelete(c, ri, ip, 1500, true) }, masterMAC)
 }
 
+func (e *ENIRoutingSuite) TestConfigureRoutewithIncompatibleIP(c *C) {
+	_, ri := getFakes(c)
+	ipv6 := net.ParseIP("fd00::2").To16()
+	c.Assert(ipv6, NotNil)
+	runConfigure(c, ri, ipv6, 1500, true)
```

Comment:
I think we can just do the call and the assertion here without modifying the helper / wrapper function (`runConfigure`) to understand different IP formats, and what to do with them.

```suggestion
	c.Assert(ri.Configure(ipv6, 1500, true), NotNil)
	// Match the error here as well
```

Reply:
Sure will do.